module2/homework/addEven/addEven.hpp

```diff
@@ -2,7 +2,11 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int suma = 0;
+
+    for (const auto & element : numbers){
+        if(element%2==0)
+        suma = suma + element;
```

Comment:
You could format your code as
`if (element % 2 == 0)` - this looks nicer and easier to read.

And it would be better to put `suma = suma + element;` in curly brackets {}. It shows explicitly what belongs to if statement. Then whole part will look like this:
```
if (element % 2 == 0) {
     suma = suma + element;
}
```